src/ocean/util/app/ext/LogExt.d

```diff
@@ -79,22 +79,38 @@ class LogExt : IConfigExtExtension
     public bool use_insert_appender;
 
 
+    /// If set, will be used by `makeLayout` to map a name to a `Layout`
+    private Appender.Layout delegate (cstring name) layout_maker;
+
+
     /***************************************************************************
 
         Constructor.
 
         Params:
+            make_layout         = A delegate that instantiate an
+                                  `Appender.Layout` from a name. If null,
+                                   defaults to `ocean.util.Config: newLayout`.
```

Comment:
Please check indentation. Also this documentation should be added to the constructor below and not here. Again the `ditto` situation...